client/src/views/pages/Auth.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+// Import components
+import Login from './Login'
+import Register from './Register'
+
+class Auth extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div>
+        <h1>Login</h1>
+        {<Login authUser={this.props.authUser} />}
```

Comment:
Why do we need to wrap this component in curly braces?

Reply:
Sorry. I had a CORS problem there. And I think I just canceled comment here. I just missed this place.
BTW, if I could solve axios local CORS conflict problem. This App could be run freely.